applications/vanilla/settings/structure.php

```diff
@@ -423,6 +428,15 @@
         ->put();
 }
 
+if (!$captureOnly) {
```

Comment:
If the is-existing check is done this far down, the columns have already been created if they didn't exist in the first place. It'll always return `true`.